core/src/main/java/org/openedx/core/UIMessage.kt

```diff
@@ -8,5 +8,7 @@ sealed class UIMessage {
         val duration: SnackbarDuration = SnackbarDuration.Long,
     ) : UIMessage()
 
+    class DatesShiftedSnackBar : UIMessage()
```

Comment:
What if we relocate class DatesShiftedSnackBar: UIMessage() within the course module? 
It appears unnecessary to inform the core module about Dates functionality.

Reply:
relocated.